### DISCUSSION TITLE
Multi Language

### DISCUSSION THREAD

**Discussion by fbenavidesd** [williamtroup/Calendar.js#148] [Q&A]
Hello, first of all, Great calendar.

Is there any example of language translation available some one can point me? So I could take as example for make my own translation to my language.

Sorry my english and thanks

**Comment by magistratus-de** [comment]
Hi,

is it possible to include the file 'dist/translations/calendar.translations.XX.js' or must the content be copied completely?

**Comment by magistratus-de** [comment]
I have tried this, but the page remains in English
```html
<html>
    <head>
        <title>TODO supply a title</title>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="stylesheet" href="dist/calendar.js.css" />
        <script src="dist/calendar.js"></script>
        <script src="dist/translations/calendar.translations.de.js"></script>
    </head>
    <body>
        <div id="calendar"></div>
        <script> 
            var calendarInstance1 = new calendarJs( "calendar", {
              manualEditingEnabled: false
            } );
        </script>
    </body>
</html>
```

**Reply by vindarel** [reply]
Same here. In [this test file](https://github.com/williamtroup/Calendar.js/blob/a9433b0526cb7cef9296f24fd0f6bb2290af5118/test/src/calendar.js.translation.html#L16) we see a line with "translation options", add writing it like this works:

            var calendarInstance = new calendarJs( "calendar", __TRANSLATION_OPTIONS );

we can see its content in the JS console.

**Comment by magistratus-de** [comment]
The language file displays the texts in the selected language. But the day is still displayed with st, nd, rd, th. 

Example: Mittwoch, 17<sup>th</sup>Januar2024
